tasks/setup_env.yml

```diff
@@ -18,6 +18,14 @@
     - ca-certificates
   when: ansible_os_family in [ "Debian" ]
 
+- name: "[ENV] - Upgrade all packages to latest version."
+  ansible.builtin.apt:
+    upgrade: "yes"
+    force_apt_get: true
+  when:
+    - ansible_os_family in [ "Debian" ]
+    - upgrade_packages_first is defined
```

Comment:
`upgrade_packages_first` is undefined variable. It should be added to defaults.

Reply:
Just looked into this and where to place. The defaults main.yml isn't appropriate. I've placed it intio os_config_ref.yml.